docs/source/entities/use-contexts.mdx

```diff
@@ -191,3 +191,35 @@ type Transaction @key(fields: "id") {
   ): Int!
 }
 ``` 
+
+## A note on resolvers
+
+Fields that are used with the context directives must be resolvable via a [reference resolver](/federation/entities/#2-define-a-reference-resolver). Query plans that fetch contextual fields are based on the entity where the `@context` directive is set. Therefore, all subgraphs that resolve that entity must support looking up their fields based on the entity's `@keys` fields.
+
+Take the [previous example](#referencing-fields-across-subgraphs). Suppose the first subgraph has a single query to retrieve the logged in user:
```

Comment:
Added this anchor, as I assumed that's the example we're talking about. Could you please confirm @clenfest ?